doc/source/deploy-guide/prepare-localhost.rst

```diff
@@ -0,0 +1,248 @@
+         project_name: cloud
+         project_domain_name: default
+         user_domain_name: ldap_users
+       identity_api_version: 3
+       cacert: /usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem
+   ansible:
+     use_hostnames: True
+     expand_hostvars: False
+     fail_on_errors: True
+
+SUSE Employees, you can access the engcloud web UI at https://engcloud.prv.suse.net/.
+For more information, see https://wiki.microfocus.net/index.php/SUSE/ECP.
+If you don’t have the SUSE root certificate installed, check
+http://ca.suse.de/ , or you can set ``insecure: True`` (not recommended).
+
+Now pre-create your environment. It is convention here to use your username
+as part of the name of objects you create.
+
+Create a keypair on engcloud (using either the horizon's web interface or
+OpenStack CLI’s ``openstack keypair create``) for accessing the
+instances created. Remember the name of this keypair (which appears as
+``foctodoodle-key`` in the example below)
+
+Set this for all the following scripts:
+
+.. code-block:: console
+
+   export OS_CLOUD=engcloud # Assuming you followed the example for clouds.yaml
+   # Your username plus whatever else you would like, will be used for naming
+   # objects you create in the cloud
+   export PREFIX=foctodoodle
+   # Set the name of the keypair you created
+   export KEYNAME=foctodoodle-key
+   # Set the name of the subnet you created
+   export INTERNAL_SUBNET=${PREFIX}-subnet
+
+Now create a network, a subnet, a router and a connection to the
+floating network (you only have to do it once):
+
+.. code-block:: console
+
+   openstack network create ${PREFIX}-net
+   openstack subnet create --network ${PREFIX}-net --subnet-range 192.168.100.0/24 ${PREFIX}-subnet
+   openstack router create ${PREFIX}-router
+   openstack router set --external-gateway floating ${PREFIX}-router
+   openstack router add subnet ${PREFIX}-router ${PREFIX}-subnet
+
+Prior to executing scripts, be aware that you may need to do some
+cleanup prior to retrying scripts or playbooks should they fail. In some
+steps a teardown script is provided to clean up any created resources.
+Reconfirming that you’ve done all the previous steps to set up now will
+save you some time later.
+
+Configure KVM deployment mechanism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The KVM support is work in progress, and the next step
+(`setup_hosts`) cannot run in KVM. Currently, for KVM, only the setup of
+airship/osh is supported. Therefore there is no extra task (other than
+setting the environment variable, see :ref:`deploymechanism`)
+required when running in your own KVM environment.
+
+Configure Ansible
+-----------------
+
+Use ARA
+~~~~~~~
+
+To use ARA, set the following environment variable before running `run.sh`.
+
+.. code-block:: console
+
+   export USE_ARA='True'
+
+To setup ARA more permanently for your user on `localhost`, create an ansible
+configuration file loading ara plugins:
```

Comment:
It might be good to mention how to use ara:
```
source ~/.socok8svenv/bin/activate
ara-manage runserver
``

Reply:
I would rather point to ARA documentation, as ARA can be used in different ways.

Reply:
True, but we're in the tips part, right? So I would expect some of the usage tips would be mentioned. (+ Pointing out documentation, true)